package.json

```diff
@@ -2,12 +2,13 @@
     "name": "@protopie/guides",
     "version": "1.0.1",
     "description": "A Vanilla Guides component that can draw ruler and manage guidelines.",
-    "main": "./dist/guides.min.js",
+    "main": "./dist/guides.js",
```

Comment:
Based on the build, prod or dev, we can have this file for both like minified but for dev, we can have source map enabled.
read the comments below

Reply:
If I make a different name of the bundle file for dev and prod versions, then during development it will be necessary to change 'main' on package.json